### PR TITLE
.gitattributes and .editorconfig

### PR DESCRIPTION
**Pull request by horst-n**
@ryancramerdesign - please refer to the single comitt messages and the provided links for more information. We need this! :-)

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -0,0 +1,2 @@
+$ cat .gitattributes
+*.module linguist-language=PHP
```
Comment (horst-n) [inline .gitattributes]: maybe, we also should add a line for ***.inc** files too?

File: wire/.editorconfig [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+root = true
+
+
+[*]
+charset = utf-8
+end_of_line = lf
+
+
+[*.{php,inc,module}]
+tab_width = 4
```
Comment (LostKobrakai) [inline wire/.editorconfig]: Do we need to specify the tab_width? I mean if it's tabs people should in theory be able to use the tab width they like.

Comment (horst-n) [inline wire/.editorconfig]: Yep, you are right. We can let tab_width out.
I think I got tricked by my editor, what shows me *.css files with tabs only 2 spaces long in PW wire directories (**without** editorconfig files!). Where my personal indentation is for all files the same (tab = 4 spaces) and thats how I have setup my editor. I think the editor derives that from top-comments spaces or something other (weird) interpretion. :-/

**Comment by ivangretsky** [comment]
It is a common practice nowadays to provide this type of things. +1 for accepting this PR.

**Comment by horst-n** [comment]
@ryancramerdesign , @isellsoap,
Anything new on this?

**Comment by isellsoap** [comment]
@horst-n Sounds good to me! I can’t merge things here, so it’s all on @ryancramerdesign. 😀

**Comment by ryancramerdesign** [comment]
Thanks Horst. I've merged this to dev. I think it's not showing a message here because I merged to dev rather than master (your PR was on master).

**Comment by horst-n** [comment]
Oh, my bad. Thanks Ryan! :)

**Pull request closed without merging**